RadarSDK/Radar.m

```diff
@@ -79,6 +79,9 @@ + (void)initializeWithPublishableKey:(NSString *)publishableKey {
                                          if (sdkConfiguration.startTrackingOnInitialize && ![RadarSettings tracking]) {
                                             [Radar startTrackingWithOptions:[RadarSettings trackingOptions]];
                                          }
+                                         if (sdkConfiguration.trackOnceOnInitialize) {
```

Comment:
how does this perform in practice? iOS does not allow us to request location multiple times at once (one of the location requests may be dropped). I think our implementation is resilient to that but lets make sure.

Reply:
for now, we keep this. there is this coalescing track feature https://github.com/radarlabs/radar-sdk-ios/pull/318 that may be used in the future.